allauth/socialaccount/providers/oauth/client.py

```diff
@@ -77,7 +77,9 @@ def _get_request_token(self):
                 raise OAuthError(
                     _('Invalid response while obtaining request token'
                       ' from "%s".') % get_token_prefix(
-                          self.request_token_url))
+                            self.request_token_url) +
+                    ' Response content: %s' % response.content.decode("utf-8")
```

Comment:
content need not be in utf-8 format, you should simply use response.text to have the decoding transparently handled for you.

Reply:
Thank you for the feedback